kuma/payments/jinja2/payments/payments.html

```diff
@@ -42,168 +42,212 @@ <h3>{{ title }}</h3>
+                    <p>
+                        {% trans %}
+                            MDN is funded out of the Mozilla Corporation general
+                            budget (and has been since it was founded in 2005).
+                            Mozilla Corporation intends to continue to financially
+                            support MDN into the future, even as we broaden and
+                            diversify the sources of MDN funding. We just want to
+                            do more things with you and for you!
+                        {% endtrans %}
+                    </p>
+                {% endcall %}
+                {%- call faq_entry(10, _("How does Mozilla make money?")) %}
+                    <p>
+                        {% trans %}
+                            The Mozilla Corporation, which funds MDN, makes money
+                            primarily from royalties from search providers on
+                            Firefox (such as Google, Amazon, Duck Duck Go, and
+                            others).
+                        {% endtrans %}
+                    </p>
+                    <p>
+                        {% trans url="https://www.mozilla.org/en-US/foundation/annualreport/2016/" %}
+                            Separately, the Mozilla Foundation is a not-for-profit,
+                            making its money primarily from donations and royalties
+                            from Mozilla Corporation. As a not-for-profit, the
+                            Mozilla Foundation reports these revenues publicly
+                            every trailing year, as in our most recent
+                            <a href="{{ url }}">2016 Mozilla Annual Report</a>.
+                        {% endtrans %}
+                    </p>
+                {% endcall %}
+
+
+            </ol>
+        </section>
+        <section class="section">
+            <div class="index">
+                <h2>{{_('Monthly Payments')}}</h2>
+            </div>
+            <ol id="contribute-monthly-faqs" class="faqs clear">
+                {%- call faq_entry(11, _("How do I manage my monthly payment?")) %}
+                    <p>
+                        {% trans mailto='mailto:%s' % settings.CONTRIBUTION_SUPPORT_EMAIL,
+                                 email=settings.CONTRIBUTION_SUPPORT_EMAIL %}
+                            If you would like to manage your monthly payment,
+                            such as changing your card account details or the amount you pay,
+                            please contact <a href="{{ mailto }}">{{ email }}</a>
+                        {% endtrans %}
+                    </p>
+                {% endcall %}
+                {%- call faq_entry(12, _("How do apply for a refund or cancel my payment?")) %}
+                    <p>
+                        {% trans mailto='mailto:%s' % settings.CONTRIBUTION_SUPPORT_EMAIL,
```

Comment:
``mailto`` to ``url``:

```suggestion
                        {% trans url='mailto:%s' % settings.CONTRIBUTION_SUPPORT_EMAIL,
```

Reply:
done